File: iniciantes/1012.py
#1012 ÁREA
valores = input().split(" ")
valoresDouble = [float(valor) for valor in valores]

pi = 3.14159

a = (valoresDouble[0] * valoresDouble[2])/2
b = (pi*(valoresDouble[2]**2))
c = ((valoresDouble[0]+valoresDouble[1])*valoresDouble[2])/2
d = valoresDouble[1]**2
e = valoresDouble[0]*valoresDouble[1]

print("TRIANGULO: {:.3f}" .format(a))
print("CIRCULO: {:.3f}" .format(b))
print("TRAPEZIO: {:.3f}" .format(c))
print("QUADRADO: {:.3f}" .format(d))
print("RETANGULO: {:.3f}" .format(e))

File: javascript/1012.js
/*Área*/
const input = require("fs").readFileSync("/dev/stdin", "utf8");

const lines = input.split(" ");

const [A, B, C] = input.split(" ");
const PI = 3.14159;

const TrianguloR = (A * C) / 2.0;
const CirculoR = PI * Math.pow(C, 2);
const Trapezio = (parseFloat(A) + parseFloat(B))*C/2.0;
const Quadrado = B*B;
const Retangulo = A*B;

console.log("TRIANGULO: " + TrianguloR.toFixed(3));
console.log("CIRCULO: " + CirculoR.toFixed(3));
console.log("TRAPEZIO: " + Trapezio.toFixed(3));
console.log("QUADRADO: " + Quadrado.toFixed(3));
console.log("RETANGULO: " + Retangulo.toFixed(3));